sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_custom_documents_async.py

```diff
@@ -69,6 +80,12 @@ async def analyze_custom_documents_async(custom_model_id):
         print("Document has type {}".format(document.doc_type))
         print("Document has document type confidence {}".format(document.confidence))
         print("Document was analyzed with model with ID {}".format(result.model_id))
+        print("...within {} bounding regions".format(format_bounding_region(document.bounding_regions)))
+        for key, value in document.fields.items():
```

Comment:
I think we should stick with `name, field` as to not confuse with key-value pairs.

Reply:
Done